lib/web_ui/lib/src/engine/compositor/util.dart

```diff
@@ -96,6 +96,70 @@ class SkiaObjects {
   }
 }
 
+/// Converts a list of [ui.Color] into the 2d array expected by CanvasKit.
+js.JsArray<Float32List> makeColorList(List<ui.Color> colors) {
```

Comment:
Issue and a TODO to make this just `Float32List`?

Reply:
Done.